scripts/leapfrog/playbooks/logstash-stop.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Stop logstash serivce
+  hosts: logstash_all
+  user: root
+  tasks:
+    - name: Service stop
+      service:
```

Comment:
Is this not covered by the existing elk upgrade process?

Reply:
Same like what I said above.